hidden-definition.html

```diff
@@ -0,0 +1,2 @@
+<p>Indicates that the <a href="#dfn-element" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-element-4">element</a> is not visible, <a href="#dfn-perceivable" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-perceivable-3">perceivable</a>, or interactive to <em>any</em> user. An element is considered <em>hidden</em> if it or any one of its ancestor elements is not rendered or is explicitly hidden.</p>
+<p>This definition includes elements <a href="https://html.spec.whatwg.org/multipage/rendering.html#being-rendered">not being rendered</a>, which don't have an associated layout box, or rendered with a <a href="https://drafts.csswg.org/css-cascade/#computed-value" data-x-internal="computed-value">computed value</a> of <a href="https://drafts.csswg.org/css2/#propdef-visibility" data-x-internal="'visibility'">'visibility'</a> different from 'visible', or hidden from assistive technologies using the <a href="https://www.w3.org/TR/wai-aria-1.2/#aria-hidden">aria-hidden</a> state.</p>
```

Comment:
The definition of hidden is from [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.2/#dfn-hidden), I'm copying it here so we can discuss.

The second paragraph is new, it specifies the different kinds of hidden referencing other definitions and other specs too. I'm using [the innerText spec](https://html.spec.whatwg.org/multipage/dom.html#the-innertext-idl-attribute) as a source for some this text, because it explains the difference between display:none and visibility:hidden in an "spec-y" way.

Reply:
Update in b2cfbb8: we discard proposing changes to WAI-ARIA definition of hidden.